src/FunctionApps/python/IntakePipeline/__init__.py

```diff
@@ -55,26 +58,39 @@ def run_pipeline(
     if bundle:
         transform_bundle(geocoder, bundle)
         add_patient_identifier(salt, bundle)
-        store_data(
-            container_url,
-            valid_output_path,
-            f"{message_mappings['filename']}.fhir",
-            message_mappings["bundle_type"],
-            bundle=bundle,
-        )
+        try:
+            store_data(
+                container_url,
+                valid_output_path,
+                f"{message_mappings['filename']}.fhir",
+                message_mappings["bundle_type"],
+                bundle=bundle,
+            )
+        except ResourceExistsError:
```

Comment:
Wouldn't we want to be able to stomp an existing resource if we're processing it differently?